conf/messages

```diff
@@ -608,6 +608,10 @@ dispatchWarehouseExcise.error.required = Enter the excise ID
 dispatchWarehouseExcise.error.length = Excise ID must be 13 characters
 dispatchWarehouseExcise.error.xss = Excise ID must not contain < and > and : and ;
 dispatchWarehouseExcise.error.format = Excise ID must start with 2 capital letters followed by 11 numbers or mixed letters and numbers
+dispatchWarehouseExcise.error.submissionError = You must update the excise ID so that it is different to the original submission
```

Comment:
```suggestion
dispatchWarehouseExcise.error.submissionError = You must update the excise ID for the dispatching tax warehouse so that it is different to the original submission
```